README.md

```diff
@@ -61,20 +61,32 @@ To learn more about the different rule types, their guarantees and how they are
 #### Carrying out software supply chain steps
 
 ##### in-toto-run
-`in-toto-run` executes the passed command and records the path and hash of the passed *materials* - files before command execution - and *products* - files after command execution and optionally stores them together with the command's *byproducts* (e.g: return value, stdout or stderr) to a link file (`<step-name>.link`), signed with the functionary's key.
+`in-toto-run` executes the passed command and records the path and hash of
```

Comment:
I found the use of the word "passed" here slightly ambiguous.  Is "given" a more effective alternative?
> in-toto-run executes the **given** command ...

It's unclear what is considered a command.   Some users might think it's a rule (e.g., in-toto-run CREATE `<pattern>`) or one of the command-line options (e.g., in-toto-run --key `<path>`), or some yet unspecified, supported command (e.g., in-toto-run start []), or some other application (e.g., in-toto-run tuf-repo --init).

Reply:
See issue #159